pkg/kubelet/kubelet_test.go

```diff
@@ -269,7 +269,7 @@ func newTestKubeletWithImageList(
 	kubelet.resyncInterval = 10 * time.Second
 	kubelet.workQueue = queue.NewBasicWorkQueue(fakeClock)
 	// Relist period does not affect the tests.
-	kubelet.pleg = pleg.NewGenericPLEG(fakeRuntime, 100, time.Hour, nil, clock.RealClock{})
+	kubelet.pleg = pleg.NewGenericPLEG(fakeRuntime, 100, time.Hour, 3*time.Minute, nil, clock.RealClock{})
```

Comment:
how about another test case on other thresholds? 3 is just default and may not be enough.

Reply:
there is one at pkg/kubelet/pleg/generic_test.go#TestHealthy